tests/Doctrine/Tests/DBAL/Platforms/AbstractPlatformTestCase.php

```diff
@@ -535,6 +535,8 @@ public function testCreateTableColumnTypeComments() : void
 
     /**
      * @return string[]
+     *
+     * @psalm-return never-return
```

Comment:
Should this be fixed in PHPUnit or in Psalm? If a function unconditionally calls a method that is marked as never-return (first option) or always throws an exception (second option), it should be automatically detected as such.

Reply:
I think I already fixed everything there is to fix in PHPUnit: https://github.com/sebastianbergmann/phpunit/pull/4187

As for Psalm, I don't know… @muglug, sorry to invoke you here, but can you please comment on this? :pray: